taccsite_cms/_settings/email.py

```diff
@@ -0,0 +1,18 @@
+"""Configure email settings (for apps e.g. djangocms_forms)"""
+
+########################
+# DJANGO
+# https://docs.djangoproject.com/en/2.2/topics/email/
+########################
+
+# Development, Staging, Production
+# https://confluence.tacc.utexas.edu/x/coR9E
+# EMAIL_HOST = "relay.tacc.utexas.edu"
+# DEFAULT_FROM_EMAIL = 'no-reply@...' # where ... is project's production domain
+
+# Debugging
+# https://docs.djangoproject.com/en/2.1/topics/email/#configuring-email-for-development
+# EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+# EMAIL_HOST = 'localhost'
+# EMAIL_PORT = 1025
+# DEFAULT_FROM_EMAIL = 'no-reply@localhost'
```

Comment:
I tested these locally. They worked. (I could see email content printed to container console output.)